### PR TITLE
Creates unit tests for TestBot

### PR DESCRIPTION
**Pull request by gabog**
Implements #2032

### CODE REVIEW

**Comment by fuselabs** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.Luis.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.QnA.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.ApplicationInsights.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Azure.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Dialogs.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.TemplateManager.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Configuration.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Connector.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Schema.dll**

**Comment by fuselabs** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.Luis.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.QnA.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.ApplicationInsights.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Azure.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Dialogs.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.TemplateManager.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Configuration.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Connector.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Schema.dll**

**Comment by fuselabs** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.Luis.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.QnA.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.ApplicationInsights.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Azure.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Dialogs.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.TemplateManager.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Configuration.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Connector.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Schema.dll**

File: libraries/Microsoft.Bot.Builder/Microsoft.Bot.Builder.csproj [hunk 1]
```diff
@@ -21,6 +21,10 @@
 		<Description>Library for building bots using Microsoft Bot Framework Connector</Description>
 		<Summary>Library for building bots using Microsoft Bot Framework Connector</Summary>
 	</PropertyGroup>
+
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Documentation|AnyCPU'">
+	  <DocumentationFile></DocumentationFile>
+	</PropertyGroup>
```
Comment (tomlm) [inline libraries/Microsoft.Bot.Builder/Microsoft.Bot.Builder.csproj]: I thought you just removed these?

File: tests/Microsoft.Bot.Builder.TestBot.Tests/stylecop.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "settings": {
+    "namingRules": {
+      "allowCommonHungarianPrefixes": true,
+      "allowedHungarianPrefixes": [
+        "lu"
```
Comment (tomlm) [inline tests/Microsoft.Bot.Builder.TestBot.Tests/stylecop.json]: > "lu" [](start = 7, length = 5)

ooooo can we add lg to this list?

Comment (gabog) [reply tests/Microsoft.Bot.Builder.TestBot.Tests/stylecop.json]: Will add it in the next PR (I got tired of stylecop complaining about Hungarian notation for lu).

File: libraries/Microsoft.Bot.Builder/Microsoft.Bot.Builder.csproj (hunk 1 above)
Comment (gabog) [reply libraries/Microsoft.Bot.Builder/Microsoft.Bot.Builder.csproj]: I did in another branch, it will all be good in master.

**Pull request merged**